### ISSUE TITLE
ctrl+n, f5 do not work

### ISSUE THREAD

**Issue by apobekiaris** [ccampbell/mousetrap#159]
```
    Mousetrap.bindGlobal('ctrl+n', function () {
        alert('Hi');
        return false;
    });
```

never called

**Comment by janmohammadi** [comment]
Doesn't work for in Chrome and Opera!

**Comment by Misiu** [comment]
This shortcut is reserved in Chrome (same as Ctrl+T), but for example Ctrl+D works fine (by default in Chrome this calls add to bookmarks dialog).
It would be nice to override Ctrl+N but I guess there isn't a workaround for now.

**Comment by ghiculescu** [comment]
@ccampbell what do you think about having Mousetrap log a warning if you try to bind a shortcut that it knows won't work? I'm thinking a short, hardcoded list - mod+n, F5, mod+t, etc.

I'm happy to write this and submit a pull request if you think it's worth doing. It would have saved me a bit of googling time if Mousetrap said "hey, that binding you just set isn't going to work, and here's why: <link to this issue>".

**Comment by kafeltz** [comment]
Isn't just a problem of e.preventDefault() would solve it?

**Comment by ghiculescu** [comment]
No. Chrome doesn't send keystroke events for these key combos at all.

**Comment by kafeltz** [comment]
What is your OS?
